framework/include/media/MediaPlayer.h

```diff
@@ -42,6 +42,8 @@ namespace media {
  */
 typedef enum player_result_e {
 	/** MediaPlayer Error case */
+	PLAYER_DEVICE_NOT_SUPPORT = -2,
+	PLAYER_INVALID_PARAM = -1,
```

Comment:
How about below?
PLAYER_ERROR_INVALID_PARAM
PLAYER_ERROR_INVALID_OPERATION
Player doesn't know about device state, so INVALID_OPERATION IS correct, I think

Reply:
That seems better, Modified.